curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md

```diff
@@ -68,13 +72,17 @@ tests:
 
 ```html
 <style>
- img {
-   max-width: 100%;
-   display: block;
-   height: auto;
- }
+.responsive-img {
+  max-width: 100%;
+  height: auto;
+}
+
+img {
+  width: 500px;
```

Comment:
```suggestion
  width: 600px;
```
Unfortunately we can't be sure it overflows, but maybe make it a little bigger to make it more obvious.